frontend/src/common/suggestitheader/suggestitheader.view.js

```diff
@@ -40,7 +41,7 @@ const SuggestITHeader = ({ renderMain }) => {
                                 })
                             }
                         />
-                        {!isLoggedIn ? (
+                        {!user ? (
```

Comment:
I would suggest using `useGamma` and `useGammaMe` hooks, along with` DigitGammaActions`. I can help you with it on the next Code & Wine

Reply:
I thought `useGamma` was depricated..
I cannot find it in the RDC story book.

Reply:
A new version of the hook was released; however, it is, as you said, not documented.

Reply:
Ok, but does that not require sharing secrets and stuff with the frontend?
And in suggestIT, I only want P.R.I.T. to be able to log in. Not anyone with a gamma account.